coral/src/app/features/approvals/connectors/components/ConnectorApprovalsTable.test.tsx

```diff
@@ -39,7 +39,7 @@ const mockedConnectorRequests: ConnectorRequest[] = [
     environmentName: "TST",
     requestor: "miketest",
     teamId: 1003,
-    teamname: "Ospo",
+    teamname: "Ospo2",
```

Comment:
Why were the team names changed in the tests? Simple curiosity.

Reply:
Same team names were resulting in failed tests as there were two elements of the same team name

Reply:
Ah yes I see!